instrumentation/aws-lambda/aws-lambda-events-2.2/javaagent/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ dependencies {
   // with lambda.
   // NB: 2.2.0 includes a class called SQSEvent but isn't usable due to it returning private classes
   // in public API.
-  library("com.amazonaws:aws-lambda-java-events:2.2.1")
+  library("com.amazonaws:aws-lambda-java-events:3.9.0")
```

Comment:
Good question, let me revert and run test target.

Reply:
Even if you revert this `javaagent` tests will still run with `3.9.0` beacuse `2.2.1` declared here is updated to  `3.9.0` because of `:instrumentation:aws-lambda:aws-lambda-events-2.2:testing`. If you make the tests run with `2.2.1` they'll fail. The way I see it there are two options, either change the supported version `2.2` -> `3.9` everywhere (docs, package name, etc.) or figure out a way how to support both versions. That could involve having a separate instrumentation for `3.9` and extracting common code so that both `3.9` and `2.2` instrumentation could use it or using reflection.